kolibri/content/utils/paths.py

```diff
@@ -1,13 +1,22 @@
 import os
+import re
 
 from django.conf import settings
+from django.core.urlresolvers import reverse
 
 try:
     from urlparse import urljoin
 except ImportError:
     from urllib.parse import urljoin
 
 
+# valid storage filenames consist of 32-char hex plus a file extension
+VALID_STORAGE_FILENAME = re.compile("[0-9a-f]{32}\.[0-9a-z]+")
```

Comment:
Fair enough. I think if we were to do it, we would want it to be based upon extensions registered by active plugins.